controller/pagecontroller.php

```diff
@@ -85,7 +85,9 @@ public function index() {
 		$params = [
 			'appName' => $appName,
 			'uploadUrl' => $this->urlGenerator->linkTo('files', 'ajax/upload.php'),
-			'publicUploadEnabled' => $this->appConfig->getAppValue('core', 'shareapi_allow_public_upload', 'yes')
+			'publicUploadEnabled' => $this->appConfig->getAppValue('core', 'shareapi_allow_public_upload', 'yes'),
```

Comment:
@oparoz Sorry, forgot to change this. I had corrected it on my local machine but forgot to update the PR